lua/resession/init.lua

```diff
@@ -352,6 +355,7 @@ M.save_all = function(opts)
       save(name, vim.tbl_extend("keep", opts, session_configs[name]), tabpage)
     end
   end
+  require("resession.util").event("SaveAllPost")
```

Comment:
Each individual session save is going to trigger a `RessionSavePost`, correct? Do we need a `ResessionSaveAllPost`?